src/mongocxx/exception/error_code.hpp

```diff
@@ -29,6 +29,7 @@ enum class error_code : std::int32_t {
     k_ssl_not_supported,
     k_unknown_read_concern,
     k_unknown_write_concern,
+    k_instance_already_exists,
```

Comment:
Per discussion, we can't do with enums (or lots of other things to) because ABI. However, we haven't set an ABI yet, so I will put it in the right place.